braindecode/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .version import __version__
 
 from .classifier import EEGClassifier
+from .regressor import EEGRegressor
```

Comment:
You should add also an entry to `docs/api.rst` file to generate documentation for this new class.

Reply:
Thanks for catching this. Will add to api.rst.